modules/unsupported/wfs-ng/src/main/java/org/geotools/data/wfs/internal/parsers/PullParserFeatureReader.java

```diff
@@ -95,6 +101,18 @@ public SimpleFeature parse() throws IOException {
         Object parsed;
         try {
             parsed = parser.parse();
+            if (parsed != null && LOGGER.isLoggable(Level.FINE)) {
```

Comment:
This logging code should probably be removed, left after debugging?

Reply:
It is left after debugging, because the original code was hanging and not showing any progress at all. This is a way to turn on a little feed-back on what's going on without interfering to much.
And the LOGGER.isLoggable(Level.FINE) makes it easy to switch off.

Reply:
It should be useful, a little off topic from the initial bug report, but useful.